deploy/operator/hypershift/playbooks/templates/infraEnv.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: agent-install.openshift.io/v1beta1
```

Comment:
I think we should have just one template of infraEnv and same goes for the other CRs
I mean, infraEnv is infraEnv no matter who's using it (maybe the template needs to be more generic, but it's up to the relevant script to fill-in all the customized values)

Reply:
Indeed, but until doing a refactor of these I prefer to keep it separated. We have lots of logic here the could be reused, and it's out of the scope...

Reply:
I think handling duplications in the playbooks / bash scripts is indeed out-of-scope here
But we can't copy-paste all those templates just because we're a little afraid to do simple moving around...

Reply:
ok, I'll open some task to handle it in a consecutive PR then.

Reply:
re-used the templates in ztp folder for now, hope it looks better.